### PR TITLE
add myanmar holidays

### PR DESCRIPTION
**Pull request by yethusoe91**

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you run the tests are your end and also include the generated snapshot in your PR?

**Comment by yethusoe91** [comment]
@freekmurze Done bro.

File: src/Countries/Myanmar.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+use Carbon\CarbonImmutable;
+
+class Myanmar extends Country
+{
+    public function countryCode(): string
+    {
+        return 'mm';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Myanmar.php:14]: this can be removed

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you rebase this branch, in order to run the tests here?

File: src/Countries/Myanmar.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+{
+    public function countryCode(): string
+    {
+        return 'mm';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return [
+            'လွတ်လပ်ရေးနေ့' => '01-04',
```
Comment (Nielsvanpach) [inline src/Countries/Myanmar.php:15]: Are all the holidays in Myanmar on a fixed date?

Comment (yethusoe91) [reply src/Countries/Myanmar.php:15]: Yes Bro. Those all dates are fixed and there will be two variable dates which may change everyyear. So skipped those days.

Comment (yethusoe91) [reply src/Countries/Myanmar.php:15]: Data Check Link

https://evisa.moip.gov.mm/home/publicholiday

Comment (Nielsvanpach) [reply src/Countries/Myanmar.php:15]: Are those variable dates based on Islamic lunar holidays?

Comment (yethusoe91) [reply src/Countries/Myanmar.php:15]: i think, No bro. 

first day is Eid al-Fitr (https://en.wikipedia.org/wiki/Eid_al-Fitr)
second day is deepavali (https://www.almanac.com/content/diwali)

Should i change day name to English bro ?

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**